help/destinations/catalog/mobile-engagement/moengage.md

```diff
@@ -0,0 +1,142 @@
+---
+keywords: mobile;mobile engagement destinations;moengage;moengage mobile engagement destination
+title: Moengage connection
+description: [!DNL Moengage]  is a customer engagement platform that powers customer-centric interactions between consumers and brands in real time.
+---
+
+# [!DNL Moengage] connection
+
+## Overview {#overview}
+
+[!DNL Moengage] destination allows you to connect and map your Adobe data (user attributes, segments and events) to MoEngage in real-time. Customers can then act on this data, delivering personalized, targeted experience.
+With Adobe, the integration is very simple and intuitive. Simply take any Adobe user profile, and map it to a MoEngage user attribute.
+
+## Use cases {#use-cases}
+
+A marketer wants to target a user segment (built in Adobe Experience Platform) via [!DNL Moengage] campaigns. Also, he wants to personalize campaign content based on attributes from the Adobe Experience Platform profiles. With this integration, users and attributes are updated in MoEngage as soon as segments and profiles are updated in Adobe Experience Platform
```

Comment:
Again minor, but I would edit "he wants to personalize" to be gender-neutral, like: "they want to personalize"

Reply:
done